mix.exs

```diff
@@ -36,10 +36,10 @@ defmodule Cereal.Mixfile do
   # Type "mix help deps" for more examples and options
   defp deps do
     [
-      {:ecto, ">= 2.0.0"},
+      {:ecto, ">= 3.0.0"},
```

Comment:
This package should work without this dependency, I only see references to structs like `Ecto.Changeset` or `Ecto.Association.NotLoaded`.